verilog/transform/obfuscate.cc

```diff
@@ -141,6 +144,38 @@ static absl::Status VerifyEquivalence(absl::string_view original,
   return absl::OkStatus();
 }
 
+// Find all modules and collect interface names
+absl::Status CollectInterfaceNames(absl::string_view content,
```

Comment:
Since this function doesn't do any "transform" operations, it would be a better fit inside `verilog/analysis`.  Maybe `verilog/analysis/extractors.cc`?